apps/browser/qml/pages/SettingsPage.qml

```diff
@@ -155,6 +155,19 @@ Page {
                 onClicked: closeAllTabsConfig.value = !closeAllTabsConfig.value
             }
 
+            TextSwitch {
+                //: Start browser in private browsing mode.
```

Comment:
Just duplicating the engineering english doesn't provide any value. Better just drop this.